toolchain/check/check.h

```diff
@@ -19,6 +20,7 @@ namespace Carbon::Check {
 // Checking information that's tracked per file.
 struct Unit {
   SharedValueStores* value_stores;
+  std::optional<Timings*> timings;
```

Comment:
```suggestion
  std::optional<Timings>* timings;
```

Preference for a pointer to the original optional, rather than a optional pointer to the object.